mmv1/products/datastream/terraform.yaml

```diff
@@ -89,3 +89,13 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           private_connection_id: "my-connection"
           network_name: "my-network"
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      constants: templates/terraform/constants/private_connection.go.erb
+      post_create: templates/terraform/post_create/private_connection.go.erb
```

Comment:
I think it could make sense to have a post_import as well - that would handle the case where the user imports something that is still `CREATING`.

Reply:
Possibly also good to check in pre-update - hypothetically it shouldn't ever hit that case but just to be safe.

Reply:
The private connection is immutable, so we cant update it (now that I think about it, not sure I mentioned it 🐒 )

Re - post import, what does import mean in this context? is that the stage when TF will read the current resource state?

Reply:
ah, yeah, it's marked as immutable in `api.yaml`, thanks!

here are the docs for [Terraform import](https://developer.hashicorp.com/terraform/cli/import) - summary is that it puts existing resources into the Terraform state - so before it's fully added to the state, we want to make sure it's finished creating.

Reply:
Added the post import method